assets/html/evolution/single_input_form.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html class="no-js" lang="en-us">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <meta name="author" content="Jing Ran">
+  <title>Single Input Form</title>
+  <meta name="description" content="{{DESCRIBE THIS PAGE HERE}}">
```

Comment:
@Jing-Ran 

You should add some description here: `{{DESCRIBE THIS PAGE HERE}}`, or at least remove it.